lookbook/docs/patterns/26-work-package-info-line.md.erb

```diff
@@ -0,0 +1,34 @@
+The work package info line is a consistent way to display the most commonly-used  set of basic work package attributes.
+
+## Overview
+
+<%= embed OpenProject::WorkPackages::InfoLineComponentPreview, :playground %>
+
+## Anatomy
+
+The info line consists of three attributes that are displayed together:
+
+- Type
+- ID
+- Status
+
+Only the ID the only interactive element; it is clickable and points the work package. All other interaction has to come from the element using this work component.
+
+## Best practices
+
+It's best to not separate these three elements in different lines. The element can, however, be wrapped in multi-line if needed.
+
+## Used in
+
+- Hovercard
+- Work package relations tab
+- Work package header
+- ...
+
+### Code structure
+
+You can typically call the InfoLineComponent in views where work package details (Type, ID, and Status) need to be displayed in one line. You must pass a work package object containing valid data to it.
+
+```html
+    render(WorkPackages::InfoLineComponent.new(work_package: @work_package))
```

Comment:
What I am actually missing as a user is to see how the component looks like. I don't know how much value it brings to explain how the component is called, as it is rather simple and very straight forward. In any case, please embedd a preview, like on [other patterns pages](https://github.com/opf/openproject/blob/dev/lookbook/docs/patterns/13-feedback-dialog-and-feedback-message.md.erb#L75).

Reply:
Hi @HDinger I added a preview in the [overview](https://github.com/opf/openproject/pull/17234/files#diff-c34c40a58d487841fe36a6a7c073d3aa157f3221a0056b45052804a97a3dbe1cR3) section.

Reply:
Ah great, I missed that 👍